website/docs/reference/events-logging.md

```diff
@@ -100,6 +182,10 @@ If available, `node_info` will include:
 
 ## Python interface
 
+:::warning
```

Comment:
Should we preserve this warning for the v1.4 docs?

Reply:
I went back and forth on this, but ultimately I agree - better safe than sorry. I switched around the version blocks so that this section keeps appearing for v1.4+